department-of-veterans-affairs/caseflow
# APPEALS-45262: Display Javascript Errors as Opposed to Cross Origin Errors

Resolves [APPEALS-45262](https://jira.devops.va.gov/browse/APPEALS-45262)

# Description
Adds 'crossorigin: anonymous' to the webpack-bundle script tag so that developers can see JS errors in development

## Acceptance Criteria
- [ ] Developers can see JS errors instead of the cross-origin error

## Testing Plan

1. Run caseflow (don't use the cold commands)
2. Navigate to something and break it: Go to the VHA decision review queue, open NonCompLayout.jsx and add `throw new Error('I am error');` somewhere
3. You should see the error screen 


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1506" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/17840614/ca6b5a61-b10c-4854-8187-87593d584fee">|<img width="1503" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/17840614/bf3f1f62-0f92-419f-9100-37e37d2f19c9">


## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
